SUTrivBot/Repo/triviaQAData.json

```diff
@@ -1,67 +1,1003 @@
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What manga did Steven have in his bedside stand during House Guest?",
+      "answerText": "Sailor Moon",
+      "answers": ["Sailor Moon"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What kind of animal steals Steven's bindle food in On the Run?",
+      "answerText": "Raccoon",
+      "answers": ["Raccoon"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What is the Steven Universe equivalent of YouTube?",
+      "answerText": "TubeTube",
+      "answers": ["TubeTube"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Which two mammals did Steven feature in a slideshow in The New Lars?",
+      "answerText": "Sloth and koala",
+      "answers": ["Sloth and koala"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What are the names of the Beetles in Giant Woman?",
+      "answerText": "Heaven and Earth",
+      "answers": ["Heaven", "Earth"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What state is Beach City located in?",
+      "answerText": "Delmarva",
+      "answers": ["Delmarva"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Where is the SU-verse's film industry located?",
+      "answerText": "Kansas",
+      "answers": ["Kansas"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What did Ronaldo call his and Lars's group in the Horror Club flashback?",
+      "answerText": "Beach City Explorer Club",
+      "answers": ["Beach City Explorer Club"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What did Pearl mistakenly call Ronaldo in Rising Tides/Crashing Skies?",
+      "answerText": "Frybo",
+      "answers": ["Frybo"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Finish the lyric: \"I bait my hook with self-pity, Someone tell me that I'm pretty...\"",
+      "answerText": "Fishin for compliments",
+      "answers": ["Fishin for compliments"],
```

Comment:
Maybe add a second possible answer "Fishin' for compliments" for us grammar nazis.